### ISSUE TITLE
The ship blocks are invisible ._.

### ISSUE THREAD

**Issue by zzx523** [ValkyrienSkies/Eureka#409]
### This issue occurs when only Valkyrien Skies and addons are installed and no other mods

- [X] I have tested this issue and it occurs when no other mods are installed

### Minecraft Version

1.19

### Eureka version

1.5.1

### VS2 version

2.1.2

### Mod Loader

Forge

### Issue description

When i click in the "Assemble" button everithing is going as it should but all ship turn invisible

### Issue reproduction

Tiny slide show below

![2024-09-30_21-20-15](https://github.com/user-attachments/assets/bf4deeea-6fc0-474f-902f-1a0ed1f81fdd)
![2024-09-30_21-21-41](https://github.com/user-attachments/assets/642bbfea-3380-4ef8-a5ca-fd8a9b178c01)
![2024-09-30_21-21-51](https://github.com/user-attachments/assets/aa3ab2ba-7628-4de7-a2c6-0757025dfd
![2024-09-30_21-21-58](https://github.com/user-attachments/assets/34840ece-0c81-470d-87d9-fce2146c73f9)
71)
![2024-09-30_21-22-10](https://github.com/user-attachments/assets/fd575197-45cd-4190-ac17-03e330bae2be)
![2024-09-30_21-23-11](https://github.com/user-attachments/assets/a33d3d69-ddfc-4b31-8fa7-fcd265bcf51e)

### Logs

[latest.log](https://github.com/user-attachments/files/17194724/latest.log)
[debug.log](https://github.com/user-attachments/files/17194726/debug.log)

**Comment by thetheaplant** [comment]
![Screenshot (84)](https://github.com/user-attachments/assets/285b9af2-b4ff-43ca-9ab7-f8c7e6777a92)
What do you think checking this box means, remove all other mods and see if the issue persists.

**Comment by zzx523** [comment]
> ![Скриншот (84)](https://private-user-images.githubusercontent.com/160092242/372202727-285b9af2-b4ff-43ca-9ab7-f8c7e6777a92.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjc4NjM1MzYsIm5iZiI6MTcyNzg2MzIzNiwicGF0aCI6Ii8xNjAwOTIyNDIvMzcyMjAyNzI3LTI4NWI5YWYyLWI0ZmYtNDNjYS05YWI3LWY4YzdlNjc3N2E5Mi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDAyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAwMlQxMDAwMzZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01OGI1ODM0ZmVhOGU4NDI0ZjJmM2U4MTUyNmJhNDZmMTkzODhlOTk5NGM4ZjI3YTUwNzlkMzgzNTI3YTg5ZGEwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.ANXfna88hzSJDAg6Sr4IZB8EQz9UxyrplAaFUClzV_c) Как вы думаете, что означает установка этого флажка? Удалите все остальные моды и проверьте, сохраняется ли проблема.

Screenshot been taken before non-mod test. I already do it, but the issue persists.

**Comment by thetheaplant** [comment]
No, it doesn't, you have optifine installed, if you removed all mods except for vs2 then the issue would not persist. Remove optifine
